css/properties/text-emphasis-position.json

```diff
@@ -60,7 +51,7 @@
             "description": "<code>left</code> and <code>right</code>",
             "support": {
               "chrome": {
-                "version_added": "62"
+                "version_added": "99"
```

Comment:
Are we not keeping data on when prefixed version was supported?

Reply:
```js
{
                  "prefix": "-webkit-",
                  "version_added": "62"
                }
```

Reply:
Oops, this change wasn't supposed to be included in this PR, reverting!